unitTests/ui/ResponsiveComponents/ResponsiveComponentsManagerTest.ts

```diff
@@ -10,6 +12,14 @@ export function ResponsiveComponentsManagerTest() {
   let responsiveComponentsManager: ResponsiveComponentsManager;
   let responsiveComponent: any;
   let component: any;
+  const setupHandleResizeEventTest = (mode?: ValidResponsiveMode) => {
```

Comment:
I think this could be `mode: ValidResponsiveMode = auto`, and then we could remove the check on line 16--18 and always assign it.

Reply:
sure :)